From: Steve Langasek <steve.langasek@ubuntu.com>
Date: Tue, 18 Feb 2020 15:43:19 +0800
Subject: avahi-dnsconfd.service: Drop "Also=avahi-daemon.socket"

'Also=avahi-daemon.socket' means that 'systemctl disable avahi-dnsconfd'
will also disable avahi-daemon.socket, which is definitely not what we
want, and it also causes debhelper to throw an error. Just drop this
entry from the configuration.

Bug-Debian: https://bugs.debian.org/878911
Applied-upstream: 0.9, commit:9c3a314856affb288f701d2d3ee23278fc98eaee
---
 avahi-dnsconfd/avahi-dnsconfd.service.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/avahi-dnsconfd/avahi-dnsconfd.service.in b/avahi-dnsconfd/avahi-dnsconfd.service.in
index 95db79f..7c293da 100644
--- a/avahi-dnsconfd/avahi-dnsconfd.service.in
+++ b/avahi-dnsconfd/avahi-dnsconfd.service.in
@@ -26,4 +26,3 @@ ExecStart=@sbindir@/avahi-dnsconfd -s
 
 [Install]
 WantedBy=multi-user.target
-Also=avahi-daemon.socket
